airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/mssql/MSSQLConverter.java

```diff
@@ -29,23 +30,28 @@ public class MSSQLConverter implements CustomConverter<SchemaBuilder, Relational
 
   private final Logger LOGGER = LoggerFactory.getLogger(MSSQLConverter.class);
 
-  private final Set<String> DATE_TYPES = Set.of("DATE", "DATETIME", "DATETIME2", "SMALLDATETIME");
   private final Set<String> BINARY = Set.of("VARBINARY", "BINARY");
+  private final Set<String> DATETIME_TYPES = Set.of("DATETIME", "DATETIME2", "SMALLDATETIME");
+  private final String DATE = "DATE";
   private static final String DATETIMEOFFSET = "DATETIMEOFFSET";
   private static final String TIME_TYPE = "TIME";
   private static final String SMALLMONEY_TYPE = "SMALLMONEY";
   private static final String GEOMETRY = "GEOMETRY";
   private static final String GEOGRAPHY = "GEOGRAPHY";
   private static final String DEBEZIUM_DATETIMEOFFSET_FORMAT = "yyyy-MM-dd HH:mm:ss XXX";
 
+  private static final String DATETIME_FORMAT_MICROSECONDS = "yyyy-MM-dd'T'HH:mm:ss[.][SSSSSS]";
```

Comment:
Why 6?

Reply:
We're hard coding it to 6 to fit with most of the popular destinations for now. This work is slated to be reverted once Destination/platform establish a precision standard on their end.